packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -112,7 +112,10 @@ export const RemoteServerFactory = {
             // be no use-case for SDK users to set this.
             strictSSL: true,
 
-            mobify: undefined
+            mobify: undefined,
+
+            // Toggle cookies being passed and set
+            ssrAllowCookies: false
```

Comment:
The impact of this setting, is that when set during local development, the development server will not strip cookies sent in HTTP requests and will not strip cookies set in HTTP responses.

I don't believe we need the `ssr` prefix here.

Is there a name we could use that suggests that this setting is only for local development and is not respected in MRT?

Reply:
`localAllowCookies` would work for me. `devAllowCookies` could work but seems less clear to me.